OrbitGl/ThreadTrack.cpp

```diff
@@ -80,6 +80,30 @@ bool ThreadTrack::IsTimerActive(const TimerInfo& timer_info) const {
   return GOrbitApp->IsFunctionVisible(timer_info.function_address());
 }
 
+[[nodiscard]] static inline Color ToColor(uint64_t val) {
+  return Color((val >> 24) & 0xFF, (val >> 16) & 0xFF, (val >> 8) & 0xFF, val & 0xFF);
+}
+
+[[nodiscard]] static inline std::optional<Color> GetUserColor(const TimerInfo& timer_info,
+                                                              const FunctionInfo* function_info) {
+  if (function_info != nullptr) {
```

Comment:
if (function_info == nullptr) {
  return {}; // Also this probably should be a check.
}

Reply:
Done.